internal/auth/authn/jwtverifer.go

```diff
@@ -0,0 +1,47 @@
+package authn
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+type JWTAuth struct {
+	JwksUrl          string
+	OidcDiscoveryUrl string
+}
+
+func (j JWTAuth) ValidateToken(ctx context.Context, token string) (bool, error) {
```

Comment:
question: what value is the bool if its always false on error?

Reply:
It must be true only in one case where the JWT token is valid. If there is an error, it should return false, which means authentication failed.

Reply:
If there is an error auth fails otherwise it passes, this is not blocking just pointing out that the bool is redundant.

Reply:
I agree, if you have suggestion to improve it let me know.